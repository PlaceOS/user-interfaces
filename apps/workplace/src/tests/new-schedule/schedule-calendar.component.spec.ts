import { createComponentFactory, Spectator } from "@ngneat/spectator/jest";
import { ScheduleCalendarComponent } from "../../app/new-schedule/schedule-calendar.component";

describe('ScheduleCalendarComponent', () => {
    let spectator: Spectator<ScheduleCalendarComponent>;
    const createComponent = createComponentFactory({
        component: ScheduleCalendarComponent
    });

    beforeEach(() => spectator = createComponent());

    it('should create component', () => {
        expect(spectator.component).toBeTruthy();
    });

    it('should match snapshot', () => {
        expect(spectator.element).toMatchSnapshot();
    })
});