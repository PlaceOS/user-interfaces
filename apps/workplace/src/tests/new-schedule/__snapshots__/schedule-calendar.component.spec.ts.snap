// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScheduleCalendarComponent should match snapshot 1`] = `
<div
  id="root1"
>
  <div
    class="p-2"
  >
    <div
      class="flex items-center justify-between"
    >
      <div
        class="px-2 font-medium"
      >
         December 
      </div>
      <div>
        <button
          mat-icon-button=""
        >
          <app-icon>
            chevron_left
          </app-icon>
        </button>
        <button
          mat-icon-button=""
        >
          <app-icon>
            chevron_right
          </app-icon>
        </button>
      </div>
    </div>
    <div
      class="flex items-center text-sm"
    >
      <div
        class="flex-1 opacity-60 text-center"
      >
         Sun 
      </div>
      <div
        class="flex-1 opacity-60 text-center"
      >
         Mon 
      </div>
      <div
        class="flex-1 opacity-60 text-center"
      >
         Tue 
      </div>
      <div
        class="flex-1 opacity-60 text-center"
      >
         Wed 
      </div>
      <div
        class="flex-1 opacity-60 text-center"
      >
         Thu 
      </div>
      <div
        class="flex-1 opacity-60 text-center"
      >
         Fri 
      </div>
      <div
        class="flex-1 opacity-60 text-center"
      >
         Sat 
      </div>
      
    </div>
    <div
      class="flex items-center flex-wrap"
    >
      <button
        class="min-w-[14%] flex-1 text-opacity-30 text-black font-normal"
        mat-icon-button=""
      >
         28 
      </button>
      <button
        class="min-w-[14%] flex-1 text-opacity-30 text-black font-normal"
        mat-icon-button=""
      >
         29 
      </button>
      <button
        class="min-w-[14%] flex-1 text-opacity-30 text-black font-normal"
        mat-icon-button=""
      >
         30 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         1 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         2 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         3 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         4 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         5 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         6 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         7 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         8 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         9 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         10 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         11 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         12 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         13 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         14 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         15 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         16 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         17 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         18 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         19 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         20 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         21 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         22 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         23 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         24 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         25 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         26 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         27 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         28 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         29 
      </button>
      <button
        class="min-w-[14%] flex-1 text-black font-normal"
        mat-icon-button=""
      >
         30 
      </button>
      <button
        class="min-w-[14%] flex-1 text-primary bg-gray-200"
        mat-icon-button=""
      >
         31 
      </button>
      <button
        class="min-w-[14%] flex-1 text-opacity-30 text-black font-normal"
        mat-icon-button=""
      >
         1 
      </button>
      <button
        class="min-w-[14%] flex-1 text-opacity-30 text-black font-normal"
        mat-icon-button=""
      >
         2 
      </button>
      <button
        class="min-w-[14%] flex-1 text-opacity-30 text-black font-normal"
        mat-icon-button=""
      >
         3 
      </button>
      <button
        class="min-w-[14%] flex-1 text-opacity-30 text-black font-normal"
        mat-icon-button=""
      >
         4 
      </button>
      <button
        class="min-w-[14%] flex-1 text-opacity-30 text-black font-normal"
        mat-icon-button=""
      >
         5 
      </button>
      <button
        class="min-w-[14%] flex-1 text-opacity-30 text-black font-normal"
        mat-icon-button=""
      >
         6 
      </button>
      <button
        class="min-w-[14%] flex-1 text-opacity-30 text-black font-normal"
        mat-icon-button=""
      >
         7 
      </button>
      <button
        class="min-w-[14%] flex-1 text-opacity-30 text-black font-normal"
        mat-icon-button=""
      >
         8 
      </button>
      
    </div>
  </div>
</div>
`;
